scripts/pre_push_hook.py

```diff
@@ -345,6 +347,22 @@ def does_diff_include_js_or_ts_files(files_to_lint):
     return False
 
 
+def does_diff_include_ts_files(files_to_lint):
```

Comment:
Already a method `does_diff_include_js_or_ts_files` follows the same convention.. Can you ptal!

Reply:
> Already a method does_diff_include_js_or_ts_files follows the same convention

Yeah, I agree but I think they are also incorrect because of the same reason!

Reply:
done